docs/containers/reference.md

```diff
@@ -499,11 +499,6 @@ For example, three templates are shown in the following example:
             "label": "Alpine-specific build command",
             "template": "docker build -p 1234:1234 -f \"${dockerfile}\" -t ${tag} \"${context}\"",
             "match": "alpine"
-        },
```

Comment:
TODO: some related changes to command customization RE: the new `${containerCommand}` placeholder.

(I'd just do them right now but the web editor isn't working at the moment)